docs/reference/filter-reference.md

```diff
@@ -389,7 +388,6 @@ Settings that are only valid when `grantType: "AuthorizationCode"`:
    * As a Kubernetes `generic` Secret, named by `secretName`/`secretNamespace`.  The Kubernetes secret must of
      the `generic` type, with the value stored under the key`oauth2-client-secret`.  If `secretNamespace` is not given, it defaults to the namespace of the Filter resource.
    * **Note**: It is invalid to set both `secret` and `secretName`.
- - `stateTTL`: (You decide this) How long the Ambassador Edge Stack will wait for the user to submit credentials to the identity provider and receive a response to that effect from the identity provider
```

Comment:
"how long" can a wait be? like 56743854398 hours? or 60 minutes? etc. Please specify the range and unit of time.

Reply:
This line is getting removed, not added.  But even if it weren't:
 - it could be any range of time; the default is specified above to be 5 minutes
 - it's stated above to be a "duration-string", which just a few lines down are explained as:

> `"duration-string"` strings are parsed as a sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". See [Go `time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration).

Reply:
Oh my bad. LGTM!